tests/test_ensemble.py

```diff
@@ -0,0 +1,122 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+Tests of Ensemble.
+
+.. codeauthor:: Oliver James Hall <ojh251@student.bham.ac.uk>
+.. codeauthor:: Rasmus Handberg <rasmush@phys.au.dk>
+.. codeauthor:: Lindsey Carboneau <lmcarboneau@gmail.com>
+
+"""
+
+from __future__ import division, print_function, with_statement, absolute_import
+import sys
+import os
+import pytest
+sys.path.append(os.path.join(os.path.dirname(__file__), '..'))
+import corrections
+import lightkurve
+import numpy as np
+INPUT_DIR = os.path.join(os.path.dirname(__file__), 'input')
+
+# TODO: Check the frange and etc algebraically
+starid = 281703991
+camera = 3
+ccd = 1
+sector =1
+#----------------------------------------------------------------------
+def test_ensemble_basics():
+	"""Check that the Ensemblecorrector can be initiated at all"""
+	with corrections.EnsembleCorrector(INPUT_DIR, plot=True) as ec:
+		assert ec.__class__.__name__ == 'EnsembleCorrector', "Did not get the correct class name back"
+		assert ec.input_folder == INPUT_DIR, "Incorrect input folder"
+		assert ec.plot == True, "Plot parameter passed appropriately"
+
+@pytest.mark.skipif(os.environ["USER"] == 'travis',
```

Comment:
Change condition to `os.environ.get('CI') == 'true' and os.environ.get('TRAVIS') == 'true'`